launch/doc/source/conf.py

```diff
@@ -35,7 +35,7 @@
 # with the same name of this package within the docs_build directory.
 # Hence we add the parent folder to the system path so that the modules from
 # this package can be imported.
-sys.path.insert(0, os.path.abspath('.'))
+sys.path.insert(0, os.path.abspath(os.path.join('..', '..', '..', '..', 'launch')))
```

Comment:
can you clarify why do you need to use the 4th upper folder ? I tried to generate the current documentation and it's working for me. 

Also the documentation online is looking fine https://docs.ros.org/en/rolling/p/launch/